resources/lang/tha.ini

```diff
@@ -0,0 +1,112 @@
+; selection
+selection.pos1.set = "ตำแหน่ง 1 ตั้งค่าให้ X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "ตำแหน่ง 2 ตั้งค่าให้ X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "ไม่มีอะไรให้เลิกทำ"
+session.undo.left = "คุณเหลือการเลิกทำอีก {%0} ครั้ง"
+session.redo.none = "ไม่มีอะไรให้ทำซ้ำ"
+session.redo.left = "คุณเหลือการทำซ้ำอีก {%0} ครั้ง"
+session.brush.added = "ได้เพิ่ม {%0} ไปยังเซสชั่น"
+session.brush.deleted = "ได้ลบ {%0} (UUID {%1})"
+session.brush.removed = "ได้นำออก {%0} (UUID {%1})"
+session.language.set = "เปลี่ยนภาษา {%0} สำเร็จ"
+session.language.notfound = "ไม่พบภาษา {%0} รีเซ็ตสู่ค่าเริ่มต้น"
+; task
+task.copy.success = "Async คัดลอกสำเร็จ, ได้ {%0}, สำเนา {%1} บล็อกจาก {%2}."
+task.count.success = "Async การวิเคราะห์สำเร็จ, ใช้ {%0}"
+task.count.result = "{%0} บล็อกที่พบในจำนวนทั้งหมด {%1} บล็อก"
+task.fill.success = "Async เติมสำเร็จ, ใช้ {%0}, {%1} บล็อกจาก {%2} การเปลี่ยนแปลง."
+task.replace.success = "Async แทนที่สำเร็จ, ใช้ {%0}, {%1} บล็อจาก {%2} การเปลี่ยนแปลง."
+task.revert.undo.success = "Async เลิกทำสำเร็จ, ใช้ {%0}, {%1} บล็อกจาก {%2} การเปลี่ยนแปลง."
+task.revert.redo.success = "Async ทำซ้ำสำเร็จ, ใช้ {%0}, {%1} บล็อกจาก {%2} การเปลี่ยนแปลง."
+; flags
+flags.keepexistingblocks = "เก็บบล็อกที่มีอยู่"
+flags.keepair = "เก็บอากาศ"
+flags.hollow = "กลวง"
+flags.hollowclosed = "กลวงพร้อมปลายปิด"
+flags.natural = "ธรรมชาติ"
+; tools
+; wand tool
+tool.wand = "Wand"
+tool.wand.lore.1 = "คลิกซ้ายที่บล็อกเพื่อกำหนดตำแหน่ง 1 ของส่วนที่เลือก"
+tool.wand.lore.2 = "คลิกขวาที่บล็อกเพื่อตั้งค่าตำแหน่งที่ 2 ของส่วนที่เลือก"
+tool.wand.lore.3 = "ใช้ //togglewand เพื่อสลับการทำงาน"
+tool.wand.disabled = "เครื่องมือ wand ถูกปิดใช้ //togglewand เพื่อเปิดการทำงานอีกครั้ง"
+tool.wand.setenabled = "เครื่องมือ wand ในขณะนี้ {%0}!"
+; debug tool
+tool.debug = "เครื่องมือดีบั๊ก"
+tool.debug.lore.1 = "คลิกซ้ายที่บล็อกเพื่อรับข้อมูล"
+tool.debug.lore.2 = "like the name and damage values of a block"
+tool.debug.lore.3 = "ใช้ //toggledebug เพื่อสลับการทำงาน"
+tool.debug.disabled = "เครื่องมือดีบั๊กถูกปิดอยู่ใช้ //toggledebug เพื่อเปิดอีกครั้ง"
+tool.debug.setenabled = "เครื่องมือดีบั๊กอยู่ในขณะนี้ {%0}!"
+; flood tool
+ui.flood.title = "Flood เมนู"
+ui.flood.options.limit = "บล็อกมากที่สุด"
+ui.flood.options.blocks = "บล็อก"
+ui.flood.options.blocks.placeholder = "บล็อกคั่นด้วยเครื่องหมายเซมิโคคอน"
+ui.flood.options.label.infoapply = "คลิกปุ่ม "ส่ง" เพื่อสมัคร"
+; brush tool
+ui.brush.title = "เมนูบรัช"
+ui.brush.content = "เมนูหลักบรัช"
+ui.brush.create = "สร้างใหม่"
+ui.brush.getsession = "เซสชั่นบรัช"
+ui.brush.edithand = "แก้ไขบรัชในมือ"
+; brush settings
+ui.brush.settings.title = "{%0} ตั้งค่าบรัช"
+; brush options
+ui.brush.options.blocks = "บล็อก"
+ui.brush.options.blocks.placeholder = "ตัวอย่าง: 1:1,2,tnt,log:12"
+ui.brush.options.diameter = "เส้นผ่าศูนย์กลาง"
+ui.brush.options.width = "ความกว้าง"
+ui.brush.options.height = "ความสูง"
+ui.brush.options.depth = "ความลึก"
+ui.brush.options.flags = "เพิ่ม flags?"
```

Comment:
As mentioned in the issue #130 and some other issues, this means "Add options", would be good if you can update this